app/customers.py

```diff
@@ -0,0 +1,51 @@
+from app.info import get_information
+from datetime import datetime
+from freezegun import freeze_time
+
+
+class Customer:
+
+    def __init__(self, dictionary: dict) -> None:
+        for key, value in dictionary.items():
+            setattr(self, key, value)
+        self.car_fuel_consumption_per_km = self.car["fuel_consumption"] / 100
+
+
+def create_class_instance_customers_list() -> list:
+    customers_list = get_information()["customers"]
+    customer_class_list = []
+    for customer in customers_list:
+        customer_class_list.append(Customer(customer))
+    return customer_class_list
+
+
+@freeze_time("01/04/2021 12:33:41")
+def make_rest_of_prints(customer: Customer, cheapest_shop: dict) -> None:
+    if customer.money < cheapest_shop["cost"]:
+        print(f"{customer.name} doesn't have "
+              f"enough money to make purchase in any shop")
+    else:
+        print(f"{customer.name} rides to {cheapest_shop['name']}")
+        customer_home_location = customer.location
+        customer.location = cheapest_shop["location"]
+
+        now = datetime.now()
+        day = now.strftime("%d/%m/%Y %H:%M:%S")
+
+        print(f"\nDate: {day}")
+        print(f"Thanks, {customer.name}, for you purchase!")
+        print("You have bought: ")
+
+        for key in customer.product_cart.keys():
+            print(f"{customer.product_cart[key]} {key}s "
+                  f"for {cheapest_shop[key]} dollars")

```

Comment:
It's better to use `dict.items()` when you use both key and value, also try to give more descriptive names instead of key and value